src/common/wifi/core/include/microsoft/net/wifi/IAccessPointController.hxx

```diff
@@ -60,6 +61,17 @@ struct IAccessPointController
      */
     virtual Ieee80211AccessPointCapabilities
     GetCapabilities() = 0;
+
+    /**
+     * @brief Set the Ieee80211 protocol of the access point.
+     *
+     * @param ieeeProtocol The Ieee80211 protocol to be set
+     * @return true
+     * @return false
+     *
+     */
+    virtual bool
+    SetIeeeProtocol(Microsoft::Net::Wifi::Ieee80211Protocol ieeeProtocol) = 0;
```

Comment:
The protocol is assumed to be IEEE 802.11 at this layer, so we should drop the `Ieee` part from the name:

```suggestion
    SetProtocol(Microsoft::Net::Wifi::Ieee80211Protocol ieeeProtocol) = 0;
```

There should be no case for ambiguity since the API layer isn't involved here.